index.html

```diff
@@ -0,0 +1,738 @@
+                  Matthew Beedham
+                </a>
+              </li>
+              <li>
+                <p>5 days ago</p>
+              </li>
+            </ul>
+          </header>
+        </article>
+        <article class="sub-article">
+          <div>
+            <h3>LATEST</h3>
+            <p>
+              <b>TNW announces Neural: A blog for human-centric AI news and analysis</b>
+            </p>
+          </div>
+          <img src="./images/neural.png" alt="robot cartoon">
+        </article>
+        <div class="sg-article-list">
+          <ul>
+            <li>PSA: Amazon Fresh grocery deliveries are now free for US Prime members</li>
+            <li>Alexis Ohanian and Serj Tankian team up to help build a social network for Armenia</li>
+            <li>Google Assistant gets its own phone number in India</li>
+            <li>Pam and Angela from The Office are re-watching the show for their new podcast</li>
+            <li>Bethesda wants you to test its game streaming tech with DOOM on iOS this year</li>
+          </ul>
+        </div>
+      </section>
+      <section>
+        <h3>Distract ></h3>
+        <article>
+          <header class="bckg-image sg-img8">
+            <h5>Most Popular</h5>
+            <h3>
+              <a href="#">
+                A massive cache of stolen OnlyFans videos have been dumped online
+              </a>
+            </h3>
+            <ul>
+              <li>
+                <a href="#">
+                  Rachel Kaser
+                </a>
+              </li>
+              <li>
+                <p>11 days ago</p>
+              </li>
+            </ul>
+          </header>
+        </article>
+        <article class="sub-article">
+          <div>
+            <h3>LATEST</h3>
+            <p>
+              <b>How to find the secret surfing game in Microsoft Edge</b>
+            </p>
+          </div>
+          <img src="./images/Microsoft-Edge-Surf-796x417.jpg" alt="Microsoft Edge surfing game">
+        </article>
+        <div class="sg-article-list">
+          <ul>
+            <li>Apple won't let villains have iPhones in movies, Star Wars director says</li>
+            <li>This Giphy/Jif peanut butter collab over GIF pronunciation is a no-brainer</li>
+            <li>Netflix added a top 10 list of its most-watched content — here’s how to find it</li>
+            <li>Hasbro announces 7 new D&D games starting with Baldur's Gate 3</li>
+            <li>Rumor: Call of Duty to launch free 'Warzone' battle royale game as early as February 18</li>
+          </ul>
+        </div>
+      </section>
+    </div>
+    <section class="latest-deals">
+      <h2>Latest deals ></h2>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img lt-dktp-img img9"></a>
+        <div class="ln-atcl-head">
+          <h3>The 2020 All-in-One Data Scientist Mega Bundle</h3>
+          <p><span class="line-through">$6000</span><span> - </span>$39.99</p>
```

Comment:
The latest deals section is not displaying the images correctly. Please fix the URL in your CSS.